internal/cmd/onepasswordtemplatefuncs.go

```diff
@@ -237,10 +257,14 @@ func (c *Config) onepasswordReadTemplateFunc(url string, args ...string) string
 	case 0:
 		// Do nothing.
 	case 1:
+		if c.Onepassword.Mode != onepasswordModeAccount {
+			panic(fmt.Errorf("1Password account parameters cannot be used in %s mode", c.Onepassword.Mode))
+		}
+
 		onepasswordArgs.account = c.onepasswordAccount(args[0])
 		onepasswordArgs.args = append(onepasswordArgs.args, "--account", onepasswordArgs.account)
 	default:
-		panic(fmt.Errorf("expected 1 or 2 arguments, got %d", len(args)))
+		panic(fmt.Errorf("expected 1 or 2 arguments, got %d", len(args)+1))
```

Comment:
Is the `+1` correct here?

Reply:
I think I added it when I was doing some testing because the output looked odd. I can try some other manual testing later to be certain.

Reply:
Yes, it is.

```console
# with +1
$ go run . execute-template '{{ onepasswordRead "a" "b" "c" }}'
chezmoi: template: arg1:1:3: executing "arg1" at <onepasswordRead "a" "b" "c">: error calling onepasswordRead: expected 1 or 2 arguments, got 3
exit status 1
# without +1
$ go run . execute-template '{{ onepasswordRead "a" "b" "c" }}'
chezmoi: template: arg1:1:3: executing "arg1" at <onepasswordRead "a" "b" "c">: error calling onepasswordRead: expected 1 or 2 arguments, got 2
exit status 1
```

For `onepasswordRead`, users won't read it as `url, ...args` but as `"a" "b" "c"`. This is a bit of a drive-by error output bug fix.